libcxx/src/CMakeLists.txt

```diff
@@ -197,7 +198,7 @@ split_list(LIBCXX_LINK_FLAGS)
 # Build the shared library.
 if (LIBCXX_ENABLE_SHARED)
   add_library(cxx_shared SHARED ${exclude_from_all} ${LIBCXX_SOURCES} ${LIBCXX_HEADERS})
-  target_include_directories(cxx_shared PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+  target_include_directories(cxx_shared PRIVATE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../../libc) #TODO: Do this properly
```

Comment:
We could do something like this instead:

```cmake
#
# runtimes/cmake/Modules/FindLibcUtils.cmake
#

# This file sets up an interface target called e.g. libc::shared-utilities that we can link against from any of the runtimes
# to get the functionality implemented in it. Concretely, we'd use it basically from libcxx/src/CMakeLists.txt as:

include(FindLibcUtils)
target_link_libraries(cxx_shared PRIVATE libc::shared-utilities) # This probably/hopefully only adds a `-I` flag.

# Possible implementation of this file:

add_library(libc::shared-utilities INTERFACE)
target_include_directories(libc::shared-utilities INTERFACE <path-to-shared-utilities>)
target_compile_features(libc::shared-utilities INTERFACE cxx_std_11) # for example
```

Reply:
I've created a file that does this, it's not the cleanest solution but it's the best I've got for the moment.